app/main.py

```diff
@@ -1 +1,36 @@
-#  write your code here
+class Animal:
+    def __init__(self, name: str, appetite: int, is_hungry: bool = True):
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self):
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self):
```

Comment:
here you need to add a check - whether the animal is hungry. And then do all the manipulations below or return 0